neo/core/event.py

```diff
@@ -256,3 +258,48 @@ def as_quantity(self):
         Return the event times as a quantities array.
         """
         return self.view(pq.Quantity)
+
+    def to_epoch(self, pairwise=False, durations=None):
+        """
+        Transform Event to Epoch.
+
+        This method has three modes of action.
```

Comment:
The suggested change just repeats the first line of the docstring.

Reply:
True, but then I would replace the first line of the doc string, to make clear that the function returns a new object based on the Event, rather than transforming the current object.

Reply:
Good point. Done